raiden/tests/fixtures/blockchain.py

```diff
@@ -96,7 +86,7 @@ def dagpath():
     return os.path.expanduser('~/.ethash/full-R23-0000000000000000')
 
 
-@pytest.fixture(scope='session', autouse=True)
+@pytest.fixture(autouse=True)
```

Comment:
Ha ... I see. I assumed this pregenerates the DAG and as such we suddenly switched to pregenerating the DAG before each test with this change.

Reading the fixture code I see that is not the case. So I made  a mini PR to rename it: https://github.com/raiden-network/raiden/pull/705